plugins/copr.py

```diff
@@ -148,6 +148,8 @@ def run(self):
 
         repo_filename = "{}/_copr_{}-{}.repo" \
                         .format(self.base.conf.get_reposdir, copr_username, copr_projectname)
+        modules_repo_filename = "/etc/fm.modules.d/_copr_{}-{}.cfg" \
```

Comment:
you could just own that directory, but question is still same... who should really own that directory and why **this** name?